src/Runtimes/Runtimes.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Appwrite\Runtimes;
+
+use Utopia\System\System;
+
+class Runtimes
+{
```

Comment:
It would have been nicer or more close to our "standard" to have all this initing process as part of the constructor, and use a `set` method to add new runtimes to the class on a private attribute `protected array $list` at the constructor level and not in `get`. This will also make the class usable if you want to manipulate it on different scopes in the future (like adding functionality for custom runtimes).

Reply:
Yes, I can do that 👍🏻